src/components/SweepFlipTransactionModa.tsx

```diff
@@ -0,0 +1,179 @@
+import { Button, Modal, Typography } from 'antd'
+import styled from 'styled-components'
+import { useReactiveVar } from '@apollo/client'
+import { CheckCircleFilled, InfoCircleFilled, LoadingOutlined } from '@ant-design/icons'
+import { ReservoirCollection } from '@appTypes/ReservoirCollection'
+import { Tokens } from '@hook/useTokens'
+import { formatToLocaleString } from 'src/utils'
+import { SweepFlipTransactionModalVar } from '@graphql/variables/TransactionVariables'
+
+import { CurrentSteps } from '@appTypes/ReservoirCurrentSteps'
+import { CollectionImage } from './shared/CollectionImage'
+
+const { Text, Title } = Typography
+
+interface SweepFlipTransactionModalProps {
+  collection?: ReservoirCollection
+  tokens: Tokens
+  totalPrice?: number
+  sweepSteps: CurrentSteps[] | undefined
+  flipSteps: CurrentSteps[] | undefined
+  onClose: () => void
+}
+
+export function SweepFlipTransactionModa({
```

Comment:
SweepFlipTransactionModaL
```suggestion
export function SweepFlipTransactionModal({
```

Reply:
Rename also file name